.github/workflows/main.yml

```
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request:
+    types: [ opened, synchronize ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  lint:
+    name: Lint
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # [[AJUSTES GERAIS]]
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.15.4'
+      ##################################
+      ##################################
+
+      # Runs a single command using the runners shell
+      - name: Node Version
+        run: node -v
+      - name: Install de dependencias
+        run: yarn install
+      - name: Roda o Linter!
+        run: yarn lint
```

O projeto utiliza NPM e não YARN
```suggestion
      - name: Install de dependencias
        run: npm install
      - name: Roda o Linter!
        run: npm run lint
```